### PR TITLE
Full Bip322 signatures

### PR DESCRIPTION
**Pull request by diehuxx**

### CODE REVIEW

**Comment by raphjaph** [comment]
I just merged #6, so that we both have consistent formatting across our editors. Could you resolve the conflicts and fix the formatting? Then I can have a look at this properly :)

**Comment by diehuxx** [comment]
@raphjaph upd8d

File: src/lib.rs [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
   },
   bitcoin_hashes::{sha256, Hash},
-  std::{io::Cursor, str},
 };
 
+mod sign;
+pub use sign::{full_sign, simple_sign};
+
+mod verifier;
+pub use verifier::{full_verify, simple_verify};
```
Comment (raphjaph) [inline src/lib.rs]: I prefer having the pub use combined and the modules underneath each other.
```suggestion
mod verifier;
pub use { 
  sign::{full_sign, simple_sign},
  verifier::{full_verify, simple_verify},
};
```

**Review by raphjaph — APPROVED** [review]
LGTM

just fixed some small nits

**Pull request merged**